daemon/inertiad/crypto/totp.go

```diff
@@ -9,25 +9,25 @@ import (
 )
 
 const (
-	totpIssuerName = "Inertia"
-	totpPeriod     = 30
-	totpSecretSize = 10
-	totpDigits     = 6
-	totpAlgorithm  = otp.AlgorithmSHA1
+	TotpIssuerName = "Inertia"
+	TotpPeriod     = 30
+	TotpSecretSize = 10
+	TotpDigits     = 6
+	TotpAlgorithm  = otp.AlgorithmSHA1
```

Comment:
in Go, uppercase first characters denote exported members, which means golint in the CI will require that these have documentation in the following format:

```go
	// TotpNoBackupCodes is the number of backup codes per user
	TotpNoBackupCodes    = 10
```

since I don't think they're used outside this package, might be best to leave them unexported for now though